src/operator.py

```diff
@@ -104,20 +106,45 @@ def is_drone(client_id):
 def is_sensor(client_id):
     '''
     Verifies that the client_id is a sensor ID or not.
-    
+
     :param client_id: id of client to identify
     '''
     if client_id % 2 == 0:
         return True
     return False
 
+def checkLatestService(request, serviceName):
+    if len(latestService) == 3 and serviceName == latestService[2] and request == latestService[0]:
+        print("Repeat service call")
+        print(latestService[1])
+        print(latestService[1]["success"])
+        return True
+    return False
+
+def saveLatestService(request, response, serviceName):
+    if len(latestService) == 0:
```

Comment:
```suggestion
    nonlocal latestService
    latestService = [request, response, serviceName]
```

Reply:
This might be a bit cleaner of a solution unless you depend on the latestService list object staying the same (mutating it rather than just straight up replacing it).